easybuild/easyconfigs/i/IPython/IPython-4.1.0-goolf-1.4.10-Python-2.7.5.eb

```diff
@@ -0,0 +1,158 @@
+    }),
+    ('tornado', '4.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/tornado/'],
+    }),
+    ('Jinja2', '2.8', {
+        'source_urls': ['https://pypi.python.org/packages/source/J/Jinja2/'],
+    }),
+    ('vcversioner', '2.14.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/v/vcversioner/'],
+    }),
+    ('jupyter_client', '4.1.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/j/jupyter_client/'],
+    }),
+    ('functools32', '3.2.3-2', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/functools32/'],
+    }),
+    ('jsonschema', '2.5.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/j/jsonschema/'],
+    }),
+    ('mistune', '0.7.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/mistune/'],
+    }),
+    ('ptyprocess', '0.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/ptyprocess/'],
+    }),
+    ('terminado', '0.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/terminado/'],
+    }),
+    ('setuptools_scm', '1.10.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/setuptools_scm/'],
+        'source_tmpl': 'setuptools_scm-1.10.1.tar.bz2',
+    }),
+    ('simplegeneric', '0.8.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/simplegeneric/'],
+        'source_tmpl': 'simplegeneric-0.8.1.zip',
+    }),
+    ('path.py', '8.1.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/path.py/'],
+        'modulename': 'path',
+    }),
+    ('ipython_genutils', '0.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/ipython_genutils/'],
+    }),
+    ('pickleshare', '0.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pickleshare/'],
+    }),
+    ('traitlets', '4.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/traitlets/'],
+    }),
+    ('pbr', '1.8.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pbr/'],
+    }),
+    ('funcsigs', '0.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/funcsigs/'],
+    }),
+    ('mock', '1.3.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/mock/'],
+    }),
+    ('notebook', '4.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/notebook/'],
+    }),
+    ('jupyter_core', '4.0.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/j/jupyter_core/'],
+    }),
+    ('ipykernel', '4.2.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/ipykernel/'],
+    }),
+    #('testpath', '0.3', {
+        #'source_urls': ['https://github.com/jupyter/testpath/archive/'], #this extension doesn't provide a setup.py and it's required for the sanity checks (iptest, iptest2)
+        #'source_tmpl': '0.3.tar.gz',
+    #}),
```

Comment:
@pescobar this is available a stand-alone easyconfig now (using `pip` to install it, which doesn't require a `setup.py`), so you can list it as a dependency rather than an extension